cdserver/app/models/building.rb

```diff
@@ -0,0 +1,8 @@
+# a building that houses shelter units
+class Building < ApplicationRecord
+  has_many :units
+
+  def bed_count
```

Comment:
ShelterBuildings also have surge beds.    Can you make that a parameter include_surge that defaults to false?
The number of surge beds available should be a parameter on the building level